Categories/Category/Instance/Groupoids.agda

```diff
@@ -0,0 +1,67 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Instance.Groupoids where
+
+-- The category of groupoids.
+--
+-- This category should maybe be called "Ho(Groupoids)" or "Ho(Gpd)"
+-- instead.  The "homsets" are not the "usual" ones consisting of
+-- functors, but consist instead of equivalence classes of functors up
+-- to natural isomorphism.  This is because homsets here are really
+-- hom-setoids and we pick natural isomorphism as the equivalence
+-- relation for these setoids.
+--
+-- See https://ncatlab.org/nlab/show/Ho%28Cat%29
+
+open import Level
+open import Categories.Category
+open import Categories.Category.IsGroupoid
+open import Categories.Functor as Fctr using (Functor; _∘F_)
+open import Categories.Functor.Properties using ([_]-resp-Iso)
+import Categories.Morphism.IsoEquiv as IsoEquiv
+open import Categories.NaturalTransformation.NaturalIsomorphism
+  using (NaturalIsomorphism; associator; unitorˡ; unitorʳ; isEquivalence; _ⓘₕ_)
+private
+  variable
+    o ℓ e : Level
+
+-- A groupoid is a category that has a groupoid structure
+
+record Groupoid (o ℓ e : Level) : Set (suc (o ⊔ ℓ ⊔ e)) where
```

Comment:
This should still be split off and put into `Categories.Category.Groupoid`.

Reply:
Hm. Should we then put `IsGroupoid` in that same module, or do you prefer to have both `Categories.Category.Groupoid` and `Categories.Category.IsGroupoid` as separate modules?

Reply:
I have no strong opinions there.  I know the stdlib splits them, but for now, I would be fine with putting them all in the same module.

Reply:
OK, I put them in the same file. Less clutter.